go.mod

```diff
@@ -216,6 +216,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+require github.com/kanisterio/safecli v0.0.6
+
```

Comment:
[nit] Can we merge this `require` block into any existing one instead of creating new?

Reply:
I would keep it in this PR since we have refs to this head PR everywhere. It looks like we have all PRs approved, so I'm going to merge this train on Monday.